README.md

```diff
@@ -17,7 +19,35 @@ _Explainer talks or demos._
 _Other writeups or less-edited notes from discussions & meetings_
 * [Are We Speced Yet?](https://docs.google.com/spreadsheets/d/1zh7Ys6Tr0y4nLsR9d9e28Q0pYNBvclmcgHU3yuwMhSI/edit?usp=sharing)
 
-### Contributing to this repo
+## Process
+We follow a process to ensure that new design docs are accessible and communicated widely.
+* Create a GitHub issue for the design doc in the appropriate repository (e.g. [go-filecoin](https://github.com/filecoin-project/go-filecoin), [consensus](https://github.com/filecoin-project/consensus)).
+* Create a new Google doc in the `Filecoin Community` team drive, `Design Docs` folder.
+  * Check that you're using the GApps identity of your choice.
+  * Use `DRAFT`, `IN REVIEW`, `IMPLEMENTING` near the top of the content to note status.
+* When you're ready, enable public comments. As of April 2019, this takes 5 clicks:
+  * Click "Share" button.
+  * Click "Who has access" to open the advanced settings panel.
+  * Toggle on "Link sharing".
+  * Choose "People at Filecoin with the link can *comment*" from dropdown.
+  * Check the checkbox for "Allow external access".
+* Add an entry with shareable link in this README's list of design docs, thus making the document public (pull request not necessary).
+* Email a link to the `dev@filecoin.org` email list, which will reach all committers. Also announce on the filecoin-project Slack.
+  * This email thread will function as as a high-signal channel for high-level discussion of the proposal.
+* Solicit feedback from individuals and/or channels, evolving the design as needed.
+  * Advertise major changes with a post to the email/slack threads where the design was announced.
+  * Seek maintainer blessing before considering the proposal accepted.
```

Comment:
Could combine 37-39 into: Solicit feedback from individuals and/or channels, evolving the design as needed. Solicit approval from [number] maintainer[s] before moving to `Accepted`.